base/setup/reactos/lang/tr-TR.rc

```diff
@@ -215,99 +214,98 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_BOOTLOADER_NOINST "No installation"
-    IDS_BOOTLOADER_REMOVABLE "Removable media"
-    IDS_BOOTLOADER_SYSTEM "System partition (Default)"
-    IDS_BOOTLOADER_MBRVBR "MBR ve VBR (Default)"
+    IDS_BOOTLOADER_NOINST "Kurulum yok"
+    IDS_BOOTLOADER_REMOVABLE "Çıkarılabilir medya"
+    IDS_BOOTLOADER_SYSTEM "Sistem bölümü (Varsayılan)"
+    IDS_BOOTLOADER_MBRVBR "MBR ve VBR (Varsayılan)"
     IDS_BOOTLOADER_VBRONLY "Yalnızca VBR"
 END
 
 /* Progress Strings */
 
 STRINGTABLE
 BEGIN
-    IDS_FORMATTING_PROGRESS1 "Formatting volume %c: (%s) in %s..."
-    IDS_FORMATTING_PROGRESS2 "Formatting volume %s in %s..."
+    IDS_FORMATTING_PROGRESS1 "%c: (%s) birimi %s dosya sisteminde biçimlendiriliyor..."
+    IDS_FORMATTING_PROGRESS2 "%s birimi %s dosya sisteminde biçimlendiriliyor..."
 
-    IDS_CHECKING_PROGRESS1 "Checking volume %c: (%s)..."
-    IDS_CHECKING_PROGRESS2 "Checking volume %s..."
+    IDS_CHECKING_PROGRESS1 "%c: (%s) birimi kontrol ediliyor..."
+    IDS_CHECKING_PROGRESS2 "%s birimi kontrol ediliyor..."
 
-    IDS_COPYING  "Copying %s"
-    IDS_MOVING   "Moving %s to %s"
-    IDS_RENAMING "Renaming %s to %s"
-    IDS_DELETING "Deleting %s"
+    IDS_COPYING  "%s kopyalanıyor"
+    IDS_MOVING   "%s dosyası %s konumuna taşınıyor"
+    IDS_RENAMING "%s dosyası %s olarak yeniden adlandırılıyor"
+    IDS_DELETING "%s siliniyor"
 
-    IDS_CONFIG_SYSTEM_PARTITION "Configuring the system partition..."
-    IDS_PREPARE_PARTITIONS "Preparing partitions..."
-    IDS_PREPARE_FILES "Preparing the list of files to be copied, please wait..."
-    IDS_COPYING_FILES "Copying the files..."
+    IDS_CONFIG_SYSTEM_PARTITION "Sistem bölümü yapılandırılıyor..."
+    IDS_PREPARE_PARTITIONS "Bölümler hazırlanıyor..."
+    IDS_PREPARE_FILES "Kopyalanacak dosyaların listesi hazırlanıyor, lütfen bekleyin..."
+    IDS_COPYING_FILES "Dosyalar kopyalanıyor..."
 
-    IDS_CREATE_REGISTRY "Creating the registry..."
-    IDS_UPDATE_REGISTRY "Updating the registry..."
+    IDS_CREATE_REGISTRY "Kayıt defteri oluşturuluyor..."
+    IDS_UPDATE_REGISTRY "Kayıt defteri güncelleniyor..."
 
-    // IDS_INSTALL_FINALIZE "Finalizing the installation..."
```

Comment:
Could you provide a translation for this string as well? It's currently commented out, but may be used in the future.

Reply:
It fixed but not shown as outdated